sklearn/gp_search.py

```diff
@@ -0,0 +1,421 @@
+"""
```

Comment:
The `grid_search` module will be `model_selection.search`, and yes, that is probably where it belongs.